massa-models/src/amount.rs

```diff
@@ -177,6 +181,45 @@ impl FromStr for Amount {
     }
 }
 
+/// Serializer for amount
+#[derive(Default)]
+pub struct AmountSerializer;
+
+impl AmountSerializer {
+    /// Create a new `AmountSerializer`
+    pub fn new() -> Self {
+        Self
+    }
+}
+
+impl Serializer<Amount> for AmountSerializer {
```

Comment:
I don't exactly understand the reason behind every `{type}Serializer` type, what prevents us to implement `Serializer` for `Amount` directly here for example?

Reply:
Because we want to be able to pass him limits or things like that and if we want to not pass it as parameters and keep it generic then we need to create a structure that holds the value.